pfrl/experiments/evaluator.py

```diff
@@ -53,13 +54,20 @@ def _run_episodes(
             test_r = 0
             episode_len = 0
             info = {}
-        a = agent.act(obs)
```

Comment:
same here. Why do you have to modify `_run_episodes` even though you have `_hrl_run_episodes`?

Reply:
same reason as above

Reply:
will change.

Reply:
will fix.